doc/sphinxman/source/postg.rst

```diff
@@ -0,0 +1,109 @@
+.. #
+.. # @BEGIN LICENSE
+.. #
+.. # Psi4: an open-source quantum chemistry software package
+.. #
+.. # Copyright (c) 2007-2019 The Psi4 Developers.
+.. #
+.. # The copyrights for code used from other parties are included in
+.. # the corresponding files.
+.. #
+.. # This file is part of Psi4.
+.. #
+.. # Psi4 is free software; you can redistribute it and/or modify
+.. # it under the terms of the GNU Lesser General Public License as published by
+.. # the Free Software Foundation, version 3.
+.. #
+.. # Psi4 is distributed in the hope that it will be useful,
+.. # but WITHOUT ANY WARRANTY; without even the implied warranty of
+.. # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.. # GNU Lesser General Public License for more details.
+.. #
+.. # You should have received a copy of the GNU Lesser General Public License along
+.. # with Psi4; if not, write to the Free Software Foundation, Inc.,
+.. # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+.. #
+.. # @END LICENSE
+.. #
+
+.. include:: autodoc_abbr_options_c.rst
+
+.. index:: postg
+.. _`sec:postg`:
+
+Interface to postg (XDM dispersion correction)
+==============================================
+
+.. codeauthor:: Alberto Otero de la Roza
+.. sectionauthor:: Alberto Otero de la Roza
+
+*Module:* :ref:`Samples <apdx:testSuitepostg>`
+
+The `exchange-hole dipole moment (XDM) model <http://schooner.chem.dal.ca/wiki/XDM>`_ 
+[Becke:2007:154108]_ is a dispersion correction used to get accurate intermolecular
+interaction energies in density-functional theory calculations. The
+XDM energy is calculated post-SCF using an external program called 
```

Comment:
This is an empirical correction, not a post-SCF a.k.a. single-shot correction, right?

Reply:
Nevermind, I see that this is ab initio but the calculation is done by postg via a Molden file, not Psi4...